Apps/JobScheduler/src/Jobs/EmailJob.cs

```diff
@@ -198,10 +198,7 @@ private void SendEmail(Email email)
             {
                 email.LastRetryDateTime = DateTime.UtcNow;
                 email.EmailStatusCode = email.Attempts < this.maxRetries ? EmailStatus.Pending : EmailStatus.Error;
-                if (caught is SmtpCommandException)
-                {
-                    email.SmtpStatusCode = (int)((SmtpCommandException)caught).StatusCode;
-                }
+                email.SmtpStatusCode = (int)((SmtpCommandException)caught).StatusCode;
```

Comment:
Why don't we have to verify this is an SmtpCommandException before casting?

Reply:
"Replace this type-check-and-cast sequence with an 'as' and a null check.", I missunderstood this fix!